contracts/ContractRegistryAccessor.sol

```diff
@@ -1,46 +1,28 @@
 pragma solidity 0.5.16;
 
 import "./spec_interfaces/IContractRegistry.sol";
-import "./spec_interfaces/IStakingContractHandler.sol";
-import "./spec_interfaces/IProtocol.sol";
-import "./spec_interfaces/ICommittee.sol";
-import "./interfaces/IElections.sol";
-import "./spec_interfaces/IGuardiansRegistration.sol";
-import "./spec_interfaces/ICertification.sol";
-import "./spec_interfaces/ISubscriptions.sol";
-import "./spec_interfaces/IDelegation.sol";
-import "./spec_interfaces/IFeesWallet.sol";
-import "./interfaces/IRewards.sol";
 import "./WithClaimableRegistryManagement.sol";
-import "./spec_interfaces/IProtocolWallet.sol";
+import "./Initializable.sol";
 
-contract ContractRegistryAccessor is WithClaimableRegistryManagement {
+contract ContractRegistryAccessor is WithClaimableRegistryManagement, Initializable {
 
-    function isManager(string memory role) internal view returns (bool) {
-        IContractRegistry _contractRegistry = contractRegistry;
-        return msg.sender == registryManager() || _contractRegistry != IContractRegistry(0) && contractRegistry.getManager(role) == msg.sender;
-    }
+    IContractRegistry contractRegistry;
 
-    modifier onlyMigrationManager {
-        require(isManager("migrationManager"), "sender is not the migration manager");
+    modifier onlyAdmin {
```

Comment:
onlyAdmin is ok, change registryManger and initializationManager to registryAdmin and initializationAdmin

Reply:
WIll rename in next PR